build/Targets/Imports.targets

```diff
@@ -520,4 +520,30 @@
     </ItemGroup>
   </Target>
 
+  <!-- 
+    Prepare inputs for Symbol Store publishing.
+     
+    The imported targets populate ArtifactsSymStoreDirectory with links to binaries and Windows PDBs produced either 
+    directly by the build or converted from Portable PDBs produced by the build.
+    In official build the content of ArtifactsSymStoreDirectory is uploaded to a symbol server.
+  -->
+  <PropertyGroup>
+    <UsingToolPdbConverter>true</UsingToolPdbConverter>
+    <PublishOutputToSymStore Condition="'$(Nonshipping)' == 'true'">false</PublishOutputToSymStore>
+    <ArtifactsSymStoreDirectory Condition="'$(DeveloperBuild)' != 'true' and '$(OS)' == 'Windows_NT'">$(BaseOutputPath)$(Configuration)\SymStore\</ArtifactsSymStoreDirectory>
+  </PropertyGroup>
+
+  <Import Project="RepoToolset\SymStore.targets" />
+
+  <Import Project="RepoToolset\SourceLink.targets" Condition="'$(EnableSourceLink)' == 'true' and '$(GitHeadSha)' != ''"/>
+
+  <PropertyGroup>
+    <PathMap Condition="'$(DeterministicSourceRoot)' != ''">$(RepoRoot)=$(DeterministicSourceRoot)</PathMap>
+  </PropertyGroup>
+
+  <Target Name="_CheckForSpaces"
```

Comment:
Nit: would give a more specific name here to avoid clashes: _CheckForSpacesInPathMap

Reply:
Fixed.

---
In reply to: [147461019](https://github.com/dotnet/roslyn/pull/22850#discussion_r147461019) [](ancestors = 147461019)